e2e/cypress/tests/integration/keyboard_shortcuts/dot_menu_spec.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @keyboard_shortcuts
+import {beUnread} from '../../support/assertions';
+import {stubClipboard} from '../../utils';
+
+describe('Keyboard Shortcuts', () => {
+    let testTeam;
+    let testChannel;
+    const postMessage = 'test for saved post';
+    const postEditMessage = ' POST EDITED';
+    before(() => {
+        // # Login as test user and visit off-topic channel
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+            cy.visit(`/${team.name}/channels/off-topic`);
+        });
+        cy.postMessage(postMessage);
+    });
+
+    it('MM-42585 Dot Menu Shorcuts', () => {
```

Comment:
[MM-T4801](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T4801) - new test case created.
```suggestion
    it('MM-T4801 Dot menu keyboard shortcuts', () => {
```